appveyor.yml

```diff
@@ -5,14 +5,15 @@ environment:
     - nodejs_version: "5"
     - nodejs_version: "6"
     - nodejs_version: "7"
+    - nodejs_version: "8"
 
 platform:
   - x86
   - x64
 
 install:
   - ps: Install-Product node $env:nodejs_version $env:platform
-  - npm install -g npm@latest
+  - npm install -g npm
```

Comment:
@juliangruber I don't know much about rabin but if it's the same setup as leveldown then you can just remove this line (so that node 0.12 uses the bundled npm)